etc/gtfs/sql.d/32-geoserver-shapes.sql

```diff
@@ -19,6 +19,10 @@ CREATE MATERIALIZED VIEW geoserver.shapes_with_routes AS
 	) t
 	GROUP BY shape_id, route_type;
 
+-- allow filtering via Geoserver's CQL
+-- todo: solve properly in gtfs-via-postgres, for all enums
+CREATE CAST (api.route_type_val AS text) WITH INOUT AS IMPLICIT;
```

Comment:
I suggest to handle an eventually pre-existing cast, to avoid issues once gtfs-via-postgres creates it itself:

```suggestion
DROP CAST IF EXISTS (api.route_type_val AS text);
CREATE CAST (api.route_type_val AS text) WITH INOUT AS IMPLICIT;
```